onnx/defs/nn/defs.cc

```diff
@@ -278,17 +281,17 @@ and computes the output.)DOC";
                         AttributeProto::INTS, OPTIONAL);
             schema.Attr("strides",
                         "stride along each axis.",
-                        AttributeProto::INTS, OPTIONAL);
+                        AttributeProto::INTS);
             schema.Attr("auto_pad",
                         auto_pad_doc.c_str(),
                         AttributeProto::STRING,
                         std::string("NOTSET"));
             schema.Attr("pads",
                         pads_doc.c_str(),
-                        AttributeProto::INTS, OPTIONAL);
+                        AttributeProto::INTS);
```

Comment:
This should be optional (default to 0s). Similar to `strides`, we need to specify the default values in the doc.